tutorials/integration.md

```diff
@@ -214,39 +239,54 @@ Using this feature is beyond the scope of this tutorial.
 
 ```
 
+&nbsp;
+
 Verify that the forwarder is the trusted forwarder for the network.
 
 ```solidity
         _verifyForwarder(relayRequest);
 ```
 
+&nbsp;
+
 This paymaster is naive, but not a complete sucker. It only accepts requests going to our 
 service. This is the way that `preRelayedCall` returns a rejection - either by
 failing a `require`, by explicitly calling `revert`, or even just running out of gas. If we return any value from this function
 normally it means that the paymaster is committed to paying for the transaction, and 
 will do so even if the transaction ultimately fails.
 
-CAUTION: Advanced Paymaster implementations may choose to override the `getGasLimits()` method of the `IPaymaster` interface.
-Doing so can create to a configuration where paymaster commits to paying for a transaction after consuming some amount of gas.
-See the [Advanced](../gsn-provider/advanced.md) section to learn more.
+{% hint style="warning" %}
+##### CAUTION: 
+Advanced Paymaster implementations may choose to override the `getGasLimits()` 
+method of the `IPaymaster` interface.
+Doing so can create to a configuration where the paymaster commits to paying for a 
+transaction after consuming some amount of gas.
+{% endhint %}
```

Comment:
Do we not have an 'advanced' section anymore? Is there a reason the link to a place I assume some more info is given removed?

Reply:
@drortirosh , do we have the advanced paymaster on the new site? I couldn't find it.